prometheus_hardware_exporter/collector.py

```diff
@@ -733,3 +735,14 @@ def fetch(self) -> List[Payload]:
     def process(self, payloads: List[Payload], datastore: Dict[str, Payload]) -> List[Payload]:
         """Process the payload if needed."""
         return payloads
+
+
+collector_registries = {
```

Comment:
This is constant, so I think it would be better to make it uppercase.

Reply:
+1

Reply:
DONE